pkg/controller/account/ec2.go

```diff
@@ -42,22 +47,35 @@ func (r *ReconcileAccount) InitializeSupportedRegions(reqLogger logr.Logger, acc
 	customerTags := r.getCustomTags(reqLogger, account)
 
 	// Create go routines to initialize regions in parallel
-
 	for _, region := range regions {
 		go r.InitializeRegion(reqLogger, account, region.Name, regionAMIs[region.Name], vCPUQuota, ec2Notifications, ec2Errors, creds, managedTags, customerTags) //nolint:errcheck // Unable to do anything with the returned error
 	}
 
+	var regionInitFailedRegion []string
+	regionInitFailed := false
 	// Wait for all go routines to send a message or error to notify that the region initialization has finished
 	for i := 0; i < len(regions); i++ {
 		select {
 		case msg := <-ec2Notifications:
 			reqLogger.Info(msg)
 		case errMsg := <-ec2Errors:
-			reqLogger.Error(errors.New(errMsg), errMsg)
+			regionInitFailed = true
+			// If we fail to initialize the desired region we want to fail the account
+			reqLogger.Error(errors.New(errMsg.ErrorMsg), errMsg.ErrorMsg)
```

Comment:
Can we make this a reqLogger.Info() call, and then add a Error() call where we're setting the account status below?

This would help keep the logs cleaner.  If we had a "Warning" level log, I'd suggest using that, but we don't have that here :/

Reply:
The region init logic is a little convoluted and I think that knowing exactly what went wrong and where is important. Also, If we don't have this here we might miss out on potential issues i.e. what if for some reason OSD accounts start failing region init but we ready the account anyways, debugging it would be a nightmare (we don't fail those accounts because they could be new - only fail CCS and BYOC on region init failure)? Since we are ignoring those failures I think these logs are even more valuable the way they currently are. In addition, I don't think we create many more AWS accounts nowadays and those are the ones that would needlessly spam the logs. What do you think? @lisa

Reply:
I agree with @ArielLima  on this one. If it's too chatty (or not enough) we can adjust it later.